db/models/storeMongo.js

```diff
@@ -1,3 +1,4 @@
+/* eslint no-console: 0 */
```

Comment:
I'm not entirely sure what this comment means, but assuming it it useful for you, theres no need to remove it.